internal/infrastructure/config/server/server.go

```
@@ -0,0 +1,61 @@
+			FileStorePath string
+			Restore       bool
+		}{}
+
+		flag.StringVar(&cfg.Address, "a", DefaultServerAddress, "server address")
+		flag.IntVar(&cfg.StoreInterval, "i", DefaultStoreInterval, "metrics store interval")
+		flag.StringVar(&cfg.FileStorePath, "f", DefaultFileStorePath, "file store path")
+		flag.BoolVar(&cfg.Restore, "r", false, "restore metrics from file")
+
+		flag.Parse()
+
+		utils.OverrideStringEnvValueWithOsEnv(&cfg.Address, "ADDRESS")
```

golangci-lint (1.62.2)

Error return value of `utils.OverrideStringEnvValueWithOsEnv` is not checked

(errcheck)